testbed/cli/main.py

```diff
@@ -42,11 +46,10 @@
     help=f"Serial port baudrate. Default: {BAUDRATE_DEFAULT}.",
 )
 @click.option(
-    "-e",
-    "--edge",
-    is_flag=True,
-    default=False,
-    help="Use MQTT adapter to communicate with an edge gateway.",
+    "-a",
+    "--adapter",
+    type=click.Choice(["serial", "mqtt"], case_sensitive=False),
```

Comment:
```suggestion
    type=click.Choice(["serial", "mqtt"], case_sensitive=False),
    default="serial",
```
This way the default choice also appears in the help message and there's no need for custom logic in the main function.